src/index.html

```diff
@@ -4,9 +4,329 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="./index.html" class="header__logo">
+        <img
+          src="./images//logo.png"
+          alt="logo"
+          class="header__logo header__logo--img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptops-&=computers"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="imac-27"
+          class="card__image"
+        >
+
+        <div class="card__description">
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <h3 class="card__subtitle">Product code: 195434</h3>
+          <div class="card__reviews">
```

Comment:
Add the blank line between the blocks
```suggestion
          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
          
          <h3 class="card__subtitle">Product code: 195434</h3>
          
          <div class="card__reviews">
```